### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by gino-m**
..PTAL?

@shobhitagarwal1612 this template adds before and after screenshot; if you agree I can add this to https://github.com/google/ground-android/blob/master/.github/pull_request_template.md as well.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Screenshot(s) before:
+
+..
+
+Screenshot(s) after:
+
+..
```
Comment (shobhitagarwal1612) [inline .github/ISSUE_TEMPLATE/pull_request_template.md:7]: These should come after PR description and verification steps

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Closes #issueno
```
Comment (shobhitagarwal1612) [inline .github/ISSUE_TEMPLATE/pull_request_template.md:9]: This will them move to the top

**Review by shobhitagarwal1612 — CHANGES_REQUESTED** [review]

**Comment by shobhitagarwal1612** [comment]
> @shobhitagarwal1612 this template adds before and after screenshot; if you agree I can add this to https://github.com/google/ground-android/blob/master/.github/pull_request_template.md as well.

Yes, SGTM

**Comment by gino-m** [comment]
@shobhitagarwal1612 RFAL!

**Review by shobhitagarwal1612 — APPROVED** [review]

**Pull request merged**